### DISCUSSION TITLE
Top 3 quantity

### DISCUSSION THREAD

**Discussion by kcasctivcr** [vdemichev/DiaNN#466] [Q&A]
Hi Vadim,

Is DIA-NN still using the sum of top 3 fragment quantity?

![image](https://user-images.githubusercontent.com/65659359/183246737-ab06b037-9e78-4f99-806b-be2eb9dd51ea.png)

The screenshot above is part of my results, seems that the first row is using the sum of top 3 fragment quantity, however the second row is not, why?

Thanks,
Hung

**Comment by vdemichev** [comment]
Hi Hung,

It's top 3 based on aerage correlation scores for the fragments. So the 3 fragments to be used are chosen for each peptide individiually.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

But how come the second row above is not the sum of the top 3?
The fragment with quantity value 22857.9 is not included in the sum, why?

Best,
Hung

**Reply by vdemichev** [reply]
Means that fragment was not included in the top 3 fragments. The fragments are selected based on all runs - not based on the specific run.

**Reply by kcasctivcr** [reply]
Hi vadim,

What does "The fragments are selected based on all runs - not based on the specific run." means?

Does this means that DIA-NN runs multiple times, and select the "best run", and for my case, the fragment with quantity value 22857.9 does not belong to the "best run"?

Many thanks,
Hung

**Reply by vdemichev** [reply]
Hi Hung, 

No, not really, please see the Nature Methods DIA-NN paper, this algorithm is explained there in detail.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

I do read the paper, but I'm still confused.
The paper states: "For each precursor, the three fragments with the highest average correlations are selected in a cross-run manner."
And in my case, only three fragments have non-zero quantity values, and these three fragments are also the only three fragments with non-zero correlation values, so how come the quantity of one of these three fragments is not included in the sum?

Best,
Hung

**Reply by vdemichev** [reply]
"For each precursor, the three fragments with the highest average correlations are selected in a cross-run manner." - means average correlations are calculated across all runs, and fragments with these being the highest are chosen. The quantities or correlations in an individual run are not the way this is decided by the algorithm.
